README.md

```diff
@@ -253,6 +254,8 @@ But if you wish to select or exclude specific linters, we give you full control
 | **VALIDATE_BASH_EXEC**             | `true`                          | Flag to enable or disable the linting process of the Bash language to validate if file is stored as executable.                                                                                                      |
 | **VALIDATE_CLOJURE**               | `true`                          | Flag to enable or disable the linting process of the Clojure language.                                                                                                                                               |
 | **VALIDATE_CLOUDFORMATION**        | `true`                          | Flag to enable or disable the linting process of the AWS Cloud Formation language.                                                                                                                                   |
+| **VALIDATE_CLOUDFORMATION_CFN_NAG** | `true`                         | Flag to enable or disbale the linting process of the AWS Cloud Formation language.
```

Comment:
Spelling:
`disable` not `disbale`

Also - maybe change it to say:

> Flag to enable or disable the security linting process of the AWS CloudFormation language.

to differentiate between this and cfn-lint.